### DISCUSSION TITLE
Thrown off by these main.cpp lines

### DISCUSSION THREAD

**Discussion by mrcrayon1** [Goldo02/gif_and_video_art_engine#13] [Q&A] [answered]
vector<int> collectionSize = {10, 4};
const vector<vector<string>> layerDir = 
{
{"1-background", "2-character", "3-head", "4-eyes", "5-lip"},
{"1-background", "2-character", "3-head"}
};

What is the second number after 10 signify? 10 = total amount of gifs to generate, 4 = ??

and why are the layers listed twice here, only on the second line the eyes and lips are skipped. 

thanks for any help

edit

sorry i realize now that the total amount generated = 14, 10 of which used all layers and 4 of which didn't use the eyes or lips.

fuckin great program here man. will tip when my project gets running.

**Comment by yaniquerobyn** [comment, marked as answer]
{"1-background", "2-character", "3-head", "4-eyes", "5-lip"},
{"1-background", "2-character", "3-head"} 

Here, there are two lines which show the names of the Layers. The first line contains 5 attributes, background, character, head eyes and lip. The second line has three Attributes, Background character and head only. 

The vector collectionSize = {10, 4}; means that it will generate 10 files using the first line of attributes and 4 files using the second line of attributes. This is good for when you have certain layers which may clash if generated together.
